incubator/elasticsearch/values.yaml

```diff
@@ -17,6 +17,9 @@ cluster:
   # If you want X-Pack installed, switch to an image that includes it, enable this option and toggle the features you want
   # enabled in the environment variables outlined in the README
   xpackEnable: false
+  # Some settings must be placed in a keystore, so they need to be mounted in from a secret.
+  # Use this setting to specify the name of the secret
+  # keystoreSecret: eskeystore
```

Comment:
this new variable should also be added to the README.md

Reply:
still missing this in README.md

Reply:
just added it